utils/computed.js

```diff
@@ -20,24 +20,24 @@ export function integerString(path) {
 /**
  * Creates a computed property that handles converting strings a list of strings that look like ['key=value'] into { key: value } and back
  * @param {*} path The path of the real value
- * @param {*} delimeter the character/s used between the key/value. Default value '='.
+ * @param {*} delimiter the character/s used between the key/value. Default value '='.
  * @returns the computed property
  */
-export function keyValueStrings(path, delimeter = '=') {
+export function keyValueStrings(path, delimiter = '=') {

```

Comment:
Will this change be fine if we're changing the name of a positional argument? 

I think it will be, but I'm leaving this note as a reminder to double-check.

Reply:
JavaScript isn't python, so there's no concept of targeting a named argument.

A method call fills argument slots, and a method definition retrieves from those slots. So this is harmless.

Effectively `delimiter` is the local alias for `arguments[1]` (`0`-based).

<details><summary>Node</summary>

```node
// Welcome to Node.js v17.3.1.
> let a=function a(b,c='2') {console.log(b,c)}
undefined
> a(1,4)
1 4
undefined
> a(1,e=4)
1 4
undefined
> (function () { console.log(arguments.length)})()
0
undefined
> (function () { console.log(arguments.length)})(1)
1
undefined
> (function () { console.log(arguments.length)})(1,2)
2
undefined
> (function () { console.log(arguments[1])})(1,2)
2
undefined
> (function () { console.log(arguments[1])})(1,3,2)
3
undefined
```
</details>